### PR TITLE
added working template of compare hours

### PR DESCRIPTION
**Pull request by peripatetic-sojourner**
Attempt at addressing #1317 and #1079.

Here's a screenshot of my proposal. I still have a lot of clean up and tests to write, but I wanted to get a concept out and see what you think.  The bottom table uses paginator and is sortable.

screenshot

### CODE REVIEW

File: assets/js/src/CompareHoursColumnChart.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    var estimated = [$("#chart").data("label-estimated")];
+    var categories = [];
+
+    for (var status in metrics) {
+        spent.push(parseInt(metrics[status].time_spent));
+        estimated.push(parseInt(metrics[status].time_estimated));
+        categories.push(status);
+    }
+
+    console.log(spent);
```
Comment (fguillot) [inline assets/js/src/CompareHoursColumnChart.js]: Don't forget to remove debug statements.

File: app/Controller/Analytic.php [hunk 2]
```diff
@@ -166,4 +167,27 @@ private function commonAggregateMetrics($template, $column, $title)
...
+            ->setOrder(TaskModel::TABLE.'.id')
+            ->setQuery($query)
+            ->calculate();
+ 
+        $stats = $this->projectAnalytic->getHoursByStatus($project['id']);
+
+        $this->response->html($this->layout('analytic/compare_hours', array(
+            'project' => $project,
+            'paginator' => $paginator,
+            'metrics' => $stats,
```
Comment (fguillot) [inline app/Controller/Analytic.php]: The title is missing here otherwise you will get a php notice.

**Comment by fguillot** [comment]
Looks good to me, if you have time to fix the comments I will be ok to merge.

Thanks.

**Comment by peripatetic-sojourner** [comment]
Made the changes. I thought you had tests for adding views, so I guess I was a lot closer to a workable solution. :smile:

**Pull request merged**